server/container/guice/jpa-guice/src/main/java/org/apache/james/JPAConfiguration.java

```diff
@@ -22,13 +22,20 @@
 import com.google.common.base.Preconditions;
 
 public class JPAConfiguration {
+
+    public static final int VALIDATION_NO_TIMEOUT = -1;
```

Comment:
Cannot you put this constant in JPA so that you don't need to define it twice?

Reply:
I can put it anywhere you want, but no one said where and I cannot find the JPA.java file you are suggesting.

Where do you want it?

Reply:
I meant in one of the parent JPA projects. As you are only making modifications to guice-jpa related, maybe james-server-jpa-common-guice would be a good candidate. But as this configuration also is globally related to jpa, puting it into a new interface in apache-james-backend-jpa project would be a good move I guess.